src/pages/signIn/index.js

```diff
@@ -35,43 +34,45 @@ const SignIn = ({ setUserPro }) => {
 
   useEffect(() => {
     if (!isNil(loginUser) && !isEmpty(loginUser)) {
-      history.push('/profile')
+      history.push(routes.profile)
     }
   }, [loginUser])
 
   return (
     <Layout>
-      <div>
-        <h1>Sign In</h1>
-        <form onSubmit={handleSubmit(onSubmit)}>
+      <div className="signInInner">
+        <h1 className="signInTitle">Sign In profile</h1>
+        <form onSubmit={handleSubmit(onSubmit)} className="formSignIn">
           <p>{error}</p>
-          <label style={{ display: 'block' }} className="inputLabel">
-            Input Email
+          <label style={{ display: 'block', marginRight: 'auto' }} className="inputLabel">
+            {!errors?.email ? 'Input email' : 'Required email'}
           </label>
-          <input
-            {...register('email', { required: 'Please input email', defaultValue: '' })}
-            type="email"
-            className="inputSign"
+          <SimpleInput
+            type={EMAIL_FIELD}
+            register={{
+              ...register(EMAIL_FIELD, { required: 'Please input email', defaultValue: '' }),
+            }}
```

Comment:
you are creating new object and destructure in it `register` result
but you could call `register` without spreading
```
register={register(EMAIL_FIELD, { required: 'Please input email', defaultValue: '' })}
```